preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,58 @@ def get_map_data():
 
     if validated is not None:
         return validated
-
+    time_stamp = int(get_timestamp())
+    name = "ca.mcgill.a11y.image.preprocessor.openstreetmap"
+    request_uuid = content["request_uuid"]
     # Check if this request is for an openstreetmap
     if 'coordinates' not in content and 'placeID' not in content:
-        logging.info("Not map content. Skipping...")
-        return "", 204
+        LOGGER.info("Not map content. Skipping...")
+        response = {
+            "request_uuid": request_uuid,
+            "timestamp": time_stamp,
+            "name": name,
+            "data": {}
+        }
+        validated = validate(
+            schema=schema,
+            data=response,
+            resolver=resolver,
+            json_message='Invalid Preprocessor JSON format',
+            error_code=500)
+
+        if validated is not None:
```

Comment:
Since this check is within the same if statement as when you declare "validated", won't this always pass? Will ever reach the the send response?

Reply:
I see that as well that other preprocessors do not return a JSON, but I did that as a preemptive move borrowing a clue from the OSM debug handler (#711, #715), where the orchestrator had to catch the returned output as an ```invalid json```, with the current format. However, this has now been slightly modified as I completely agree with you that it might never reach the send response.